installers/Windows/README.md

```diff
@@ -41,4 +41,56 @@ https://github.com/mu-editor/mu/blob/master/win_installer.py
 
 Files covered:
 
-installer.py
\ No newline at end of file
+installer.py
+
+## Step by Step: Creating a Spyder Windows Installer
+To create a Spyder Windows Installer you must follow the following steps.
+
+### Creating a Conda Environment 
+- Download and install Anaconda. You can find the latest version [here](https://www.anaconda.com/products/distribution). 
+- Open the Anaconda prompt and make sure that you have admin permissions.
+- Create the new enviroment with the following commands:
+bash
+$ conda create -n installer python=3.8.10
+$ conda activate installer
+
+The first command creates an enviroment called "installer" with Python version 3.8.10 and the second one, changes the current enviroment to the new one, "installer".'
+
+After changing to the new enviroment, install yarg with pip. 
+bash
+pip install yarg
+
+### Creating Enviroment Variables
+Before running any script, you must set some enviroment variables first. 
+The list below shows all the variables that must be created.
```

Comment:
We could a description of each of these env variables following the comments at the `installer.py` file: https://github.com/spyder-ide/spyder/blob/e63df48d249582edba32aba9fe8c98a4d773a9fe/installers/Windows/installer.py#L38-L71